doc/presentations.rst

```diff
@@ -24,21 +52,44 @@ specific subject areas:
 Videos
 ======
 
-- An introduction to scikit-learn `Part
-  I <https://conference.scipy.org/scipy2013/tutorial_detail.php?id=107>`_ and
-  `Part II <https://conference.scipy.org/scipy2013/tutorial_detail.php?id=111>`_ at Scipy 2013
-  by `Gael Varoquaux`_, `Jake Vanderplas`_  and `Olivier Grisel`_. Notebooks on
+- The `:probabl YouTube channel <https://www.youtube.com/@probabl_ai>`_
+  hosts the Whiteboard Series, which provides supplementary materials for
+  the scikit-learn MOOC and covers more recent versions of the library:
```

Comment:
Sorry I did not express myself correctly. The mooc also covers up to the latest release, I meant this playlist covers recent versions of the library w.r.t. the other videos listed in this section.
```suggestion
  the scikit-learn MOOC and covers more features of the library:
```